diff --git a/node_modules/expo-audio/ios/AudioModule.swift b/node_modules/expo-audio/ios/AudioModule.swift
index e0f8ac5..4df60ad 100644
--- a/node_modules/expo-audio/ios/AudioModule.swift
+++ b/node_modules/expo-audio/ios/AudioModule.swift
@@ -498,7 +498,6 @@ public class AudioModule: Module {
       let hasActivePlayers = self.registry.allPlayers.values.contains { $0.isPlaying }
       if !hasActivePlayers {
         do {
-          try AVAudioSession.sharedInstance().setActive(false, options: [.notifyOthersOnDeactivation])
         } catch {
           print("Failed to deactivate audio session: \(error)")
         }
